book/src/keymap.md

```diff
@@ -45,40 +45,42 @@
-| `c`         | Change selection (delete and enter insert mode) | `change_selection`    |
+| Key         | Description                                                | Command                   |
+| -----       | -----------                                                | -------                   |
+| `r`         | Replace with a character                                   | `replace`                 |
+| `R`         | Replace with yanked text                                   | `replace_with_yanked`     |
+| `~`         | Switch case of the selected text                           | `switch_case`             |
+| `` ` ``     | Set the selected text to lower case                        | `switch_to_lowercase`     |
+| `` Alt-` `` | Set the selected text to upper case                        | `switch_to_uppercase`     |
+| `i`         | Insert before selection                                    | `insert_mode`             |
+| `a`         | Insert after selection (append)                            | `append_mode`             |
+| `I`         | Insert at the start of the line                            | `prepend_to_line`         |
+| `A`         | Insert at the end of the line                              | `append_to_line`          |
+| `o`         | Open new line below selection                              | `open_below`              |
+| `O`         | Open new line above selection                              | `open_above`              |
+| `.`         | Repeat last change                                         | N/A                       |
+| `u`         | Undo change                                                | `undo`                    |
+| `U`         | Redo change                                                | `redo`                    |
+| `y`         | Yank selection                                             | `yank`                    |
+| `p`         | Paste after selection                                      | `paste_after`             |
+| `P`         | Paste before selection                                     | `paste_before`            |
+| `"` `<reg>` | Select a register to yank to or paste from                 | `select_register`         |
+| `>`         | Indent selection                                           | `indent`                  |
+| `<`         | Unindent selection                                         | `unindent`                |
+| `=`         | Format selection (**LSP**)                                 | `format_selections`       |
+| `d`         | Delete selection                                           | `delete_selection`        |
+| `Alt-d`     | Delete selection (dont yank)                               | `delete_selection_noyank` |
+| `c`         | Change selection (delete and enter insert mode)            | `change_selection`        |
+| `Alt-c`     | Change selection (delete and enter insert mode, dont yank) | `change_selection_noyank` |
```

Comment:
```suggestion
| Key         | Description                                                 | Command                   |
| -----       | -----------                                                 | -------                   |
| `r`         | Replace with a character                                    | `replace`                 |
| `R`         | Replace with yanked text                                    | `replace_with_yanked`     |
| `~`         | Switch case of the selected text                            | `switch_case`             |
| `` ` ``     | Set the selected text to lower case                         | `switch_to_lowercase`     |
| `` Alt-` `` | Set the selected text to upper case                         | `switch_to_uppercase`     |
| `i`         | Insert before selection                                     | `insert_mode`             |
| `a`         | Insert after selection (append)                             | `append_mode`             |
| `I`         | Insert at the start of the line                             | `prepend_to_line`         |
| `A`         | Insert at the end of the line                               | `append_to_line`          |
| `o`         | Open new line below selection                               | `open_below`              |
| `O`         | Open new line above selection                               | `open_above`              |
| `.`         | Repeat last change                                          | N/A                       |
| `u`         | Undo change                                                 | `undo`                    |
| `U`         | Redo change                                                 | `redo`                    |
| `y`         | Yank selection                                              | `yank`                    |
| `p`         | Paste after selection                                       | `paste_after`             |
| `P`         | Paste before selection                                      | `paste_before`            |
| `"` `<reg>` | Select a register to yank to or paste from                  | `select_register`         |
| `>`         | Indent selection                                            | `indent`                  |
| `<`         | Unindent selection                                          | `unindent`                |
| `=`         | Format selection (**LSP**)                                  | `format_selections`       |
| `d`         | Delete selection                                            | `delete_selection`        |
| `Alt-d`     | Delete selection (don't yank)                               | `delete_selection_noyank` |
| `c`         | Change selection (delete and enter insert mode)             | `change_selection`        |
| `Alt-c`     | Change selection (delete and enter insert mode, don't yank) | `change_selection_noyank` |
```

Reply:
(This just adds apostrophes and makes the whole table 1 column wider to accommodate).